dash/backend/src/modules/metrics/services/m9sweeper_cron_jobs.service.ts

```diff
@@ -1,41 +1,52 @@
 import {Injectable} from "@nestjs/common";
-import {PrometheusService} from "../modules/shared/services/prometheus.service";
-import {ImageService} from "../modules/image/services/image.service";
+import {PrometheusV1Service} from "./prometheus-v1.service";
+import {ImageService} from "../../image/services/image.service";
 import {Cron} from "@nestjs/schedule";
-import {ExceptionsService} from "../modules/exceptions/services/exceptions.service";
+import {ExceptionsService} from "../../exceptions/services/exceptions.service";
 import {formatDistance} from "date-fns";
+import { MineLoggerService } from "../../shared/services/mine-logger.service";
+
 
 @Injectable()
 export class M9sweeperCronJobService {
     constructor(
-        private readonly prometheusService: PrometheusService,
+        private readonly loggerService: MineLoggerService,
+        private readonly prometheusService: PrometheusV1Service,
         private readonly imageService: ImageService,
         private readonly exceptionsService: ExceptionsService,
     ) {}
 
     /**
-     * Recompile M9sweeper Metrics every 5 minutes for Prometheus metrics
+     * Recompile M9sweeper Metrics every 30s for Prometheus metrics
      */
-    @Cron('0 */5 * * * *')
+    @Cron('* * 01 * * *')
```

Comment:
I think we discussed getting rid of this in leau of the new metrics API you are using. 

So then we'd have 2 metrics APIs: 
 - one that provides instance-level metrics (cpu usage, ram usage, response times, request volume, error rate)
 - One that provides env level details

So then this one would be supplanted by the new one that provides env level details.

Reply:
Yes, that's the plan in the next PR :)

Reply:
(Sorry in this PR not the next one - this is just a work in progress)